basicsr/models/video_base_model.py

```diff
@@ -34,10 +34,9 @@ def dist_validation(self, dataloader, current_iter, tb_logger, save_img):
                     len(self.opt['val']['metrics']),
                     dtype=torch.float32,
                     device='cuda')
-
+            for _, tensor in self.metric_results.items():
```

Comment:
1. The program only enters L28 at the beginning of one experiment run, while L37 needs to executed when calling dist_validation in each validation phase. So this modification is not right.
2. I recommend adding an extra `if clause`  in previous version.

Reply:
Thank you for your advice and I' ll check about that.